bittensor/__init__.py

```diff
@@ -1,21 +1,20 @@
 # The MIT License (MIT)
-# Copyright © 2021 Yuma Rao
-# Copyright © 2022-2023 Opentensor Foundation
-# Copyright © 2023 Opentensor Technologies Inc
-
+# Copyright © 2024 OpenTensor Foundation
```

Comment:
Opentensor not OpenTensor

Reply:
This license has been through so many reviews. No one noticed before.